requirements/edx/github.txt

```diff
@@ -101,7 +101,7 @@ git+https://github.com/edx/help-tokens.git@v1.0.0#egg=help-tokens==1.0.0
 
 # Third Party XBlocks
 
-git+https://github.com/proversity-org/edx-sga@v0.8.2#egg=edx-sga==0.8.2
+git+https://github.com/proversity-org/edx-sga@459cd607084d78a531ca2927e4f93a9cc3e026bb#egg=edx-sga==0.8.2.1
```

Comment:
Lets try to maintain version notation as it is. Use something like `0.8.3`

Reply:
Ok @nicovanniekerk let us know then! thanks

Reply:
@diegomillan sorry i am a bit swamped with the server outages and so on, i made a new Pr on the SGA xblock to merge the and/sga_teams branch into master, but it required a configuration set that i dont know what it needs to be as well as one change request. can @andrey-canon please look at that PR make the changes and then once we merge that pr to master we can remake its 0.8.3 tag and update this PR. sorry for not finding time to do this.